service/src/main/java/com/epam/esm/validator/GiftCertificateValidator.java

```diff
@@ -19,62 +20,60 @@ public final class GiftCertificateValidator {
     private static final int MAX_DURATION = 366;
     private static final int MIN_DURATION = 1;
 
-    private GiftCertificateValidator() {
-    }
-
-    public static void validate(GiftCertificate giftCertificate) throws IncorrectParameterException {
-        validateName(giftCertificate.getName());
-        validateDescription(giftCertificate.getDescription());
-        validatePrice(giftCertificate.getPrice());
-        validateDuration(giftCertificate.getDuration());
-        validateListOfTags(giftCertificate.getTags());
+    public static void validate(GiftCertificate giftCertificate, ExceptionResult er) {
```

Comment:
You may use spring boot validation, instead of writing additional code like that. Try to learn and use it for validation